webapp/src/components/conference/conference.tsx

```diff
@@ -326,7 +334,9 @@ export default class Conference extends React.PureComponent<Props, State> {
         return (
             <React.Fragment>
                 <div
+                    onChange={this.escFunction}
```

Comment:
Is this needed? When would it be triggered?

Reply:
removed..